packages/zowe-explorer/src/dataset/ZoweDatasetNode.ts

```diff
@@ -449,6 +450,17 @@ export class ZoweDatasetNode extends ZoweTreeNode implements IZoweDatasetTreeNod
         return this.etag;
     }
 
+    /**
+     * Returns the imperative.IProfileLoaded profile for this node
+     *
+     * @returns {imperative.IProfileLoaded}
+     */
+    // public getProfile(): imperative.IProfileLoaded {
+    //     ZoweLogger.trace("ZoweDatasetNode.getProfile called.");
+    //     const prof = this.profile ?? this.getParent()?.getProfile();
+    //     return prof.name ? Profiles.getInstance().loadNamedProfile(prof.name) : undefined; // this returns the profile with newer token
+    // }
```

Comment:
Hey @KaranP25,
Was this commented out by mistake? 😋

Reply:
Oops shouldn't be like that, let me take a look

Reply:
I have updated this to include the method